rules/order.js

```diff
@@ -102,35 +176,56 @@ const order = [
     'font',
     'font-family',
     'font-variant',
+    'font-variant-alternates',
+    'font-variant-caps',
+    'font-variant-east-asian',
+    'font-variant-ligatures',
+    'font-variant-numeric',
+    'font-variant-position',
     'font-size',
     'font-size-adjust',
     'font-kerning',
     'font-weight',
     'font-stretch',
     'font-style',
     'font-smoothing',
```

Comment:
I think we have a typo here, should be `font-smooth`.

Actually it was like that because they previously worked around `vendor prefixes` but not anymore. Therefore, we need to also add `-webkit-font-smoothing` and `-moz-osx-font-smoothing` below `font-smooth`.